lib/tessel/rsa-delegation.js

```diff
@@ -0,0 +1,5 @@
+function MockRSA() {}
+
+MockRSA.prototype.exportKey = function() {};
+
+module.exports = global.IS_TEST_ENV ? MockRSA : require('node-rsa');
```

Comment:
This allows us to mock RSA key creation in the tests. Previously, `new NodeRSA(...)` was the longest running operation in the test suite, and was run several times by the provision and key tests